File: src/main/java/score/School.java
package score;

import org.jetbrains.annotations.NotNull;

import java.util.Scanner;

public class School {
    public static void main(String[] args) {
//        Student student = userInput();
//        Student.pass = 60;
        Student studentA = new Student("Rubyyyyer", "1200", 90, 99);
        Student studentB = new Student("Ericcccc","1201",60,90);
        Student studentC = new Student("Janeeee","1300",40,20);
        GraduateStudent graduateStudentA = new GraduateStudent("Selinnnn","1368",60,60,100);
        studentA.showScore();
        studentB.showScore();
        studentC.showScore();
        graduateStudentA.showScore();

    }

    @NotNull
    private static Student userInput() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("name:");
        String name = scanner.next();
        Integer idNum = (int) Math.round(Math.random() * 1000);
        String id = Integer.toString(idNum);
        System.out.println(id);
        System.out.print("eng score?");
        int eng = scanner.nextInt();
        System.out.print("math score?");
        int math = scanner.nextInt();

        Student student = new Student(name, id, eng, math);

        return student;
    }
}

File: src/main/kotlin/scoreKt/School.kt
package scoreKt

import java.util.Scanner
import kotlin.random.Random

fun main() {
//    userInput()
//    Student.pass = 80
    val studentA = Student("Ruby", "1200",70,80)
    val studentB = Student("Eric","1201",90,20)
    val studentC = Student("Jane","1222",60,60)
    val graduateStudentA = GraduateStudent("Testttt","1333",70,98,40)
    val graduateStudentB = GraduateStudent("Wooooo","1450",80,98,60)
    studentA.showScore()
    studentB.showScore()
    studentC.showScore()
    graduateStudentA.showScore()
    graduateStudentB.showScore()

}

private fun userInput() {
    val scanner = Scanner(System.`in`)
    print("name:")
    var name = scanner.next()
    val id = Random.nextInt(0, 1001).toString()
//    val id = (0..1000).random().toString()
    println(id)
    print("eng?")
    var eng = scanner.nextInt()
    print("math?")
    var math = scanner.nextInt()
    Student(name, id, eng, math).showScore()
}